packages/contracts-bedrock/deploy/021-SystemDictatorSteps-2.ts

```diff
@@ -319,29 +313,16 @@ const deployFn: DeployFunction = async (hre) => {
       30000,
       1000
     )
-  }
 
-  // Step 6 unpauses the new L1CrossDomainMessenger.
-  await doStep({
-    isLiveDeployer,
-    SystemDictator,
-    step: 6,
-    message: `
-      Step 6 will unpause the new L1CrossDomainMessenger. After this step is executed, users will
-      be able to deposit and withdraw assets via the L1CrossDomainMessenger and the system will be
-      fully operational.
-    `,
-    checks: async () => {
-      await assertContractVariable(L1CrossDomainMessenger, 'paused', false)
-    },
-  })
+    await assertContractVariable(OptimismPortal, 'paused', false)
+  }
 
   // At the end we finalize the upgrade.
-  if (await isStep(SystemDictator, 7)) {
+  if (await isStep(SystemDictator, 6)) {
```

Comment:
This if statement and the above can be merged- both rely on the current `SystemDictator` step being `6`.